docs/GettingStarted/configuration/ExporterFailure.md

```diff
@@ -366,4 +384,24 @@ A custom field can be configure with the following steps:
 
 - Navigate to an existing Incident.
 - Use the upper left Menu and select Configure -> Form Layout.
-- Create a new field (String, Table or reference a List).
\ No newline at end of file
+- Create a new field (String, Table or reference a List).
+
+## Configuring PagerDuty
+
+### Default workflow
+
+By default, Failure Time Exporter(s) configured to work with PagerDuty will:
+
+* Monitor all incidents in the domain of the token used to access it (PagerDuty's token handles all the information about credentials, including their URL, without using `SERVER`).
```

Comment:
I did this because of the sprint requirements https://github.com/orgs/konveyor/projects/51/views/2?pane=issue&itemId=20613492

But we can change it. Do you suggest anything?

Reply:
By default, Failure Time Exporter(s) configured to work with PagerDuty will:

* Monitor all incidents in the domain of the token used to access it (The API token from PagerDuty manages both the API URL endpoint and the credentials information.)